### DISCUSSION TITLE
Gallery of examples?

### DISCUSSION THREAD

**Discussion by jason-sachs** [cetz-package/cetz#622] [Q&A]
Is there a gallery of CeTZ examples? I can't seem to find one and I don't have Discord access while at work.

I'm specifically interested in any block diagrams.

**Comment by johannes-wolf** [comment]
We currently only have the examples on the GitHub page.
For block diagrams, check out Fletcher: https://github.com/Jollywatt/typst-fletcher, which also has some examples.
